### DISCUSSION TITLE
Wyze wall switches support

### DISCUSSION THREAD

**Discussion by paulmorgantx** [jfarmer08/homebridge-wyze-smart-home#72] [Ideas]
Hi,

Thank you for your plug-in. It supports the biggest kinds of Wyze accessories, but, unfortunately, looks like it doesn't support wall switches yet. Do you have plans to implement this option as well?

Thanks!

**Comment by jfarmer08** [comment]
This is correctly in the dev branch and beta branch. 

Beta branch adds support for wall switches. 

Dev branch is just that. Currently has wall switch, hms, and thermostat support. 

Dev will soon go into beta.

**Comment by paulmorgantx** [comment]
That's awesome. I've checked the version to Dev and Beta and logs didn't show any connections to switches. I've switched back to the latest version and I will look forward to releases. 

Thank you!
